db/migrate/20230608145455_add_personal_statement_to_application_choice.rb

```diff
@@ -0,0 +1,5 @@
+class AddPersonalStatementToApplicationChoice < ActiveRecord::Migration[7.0]
+  def change
+    add_column(:application_choices, :personal_statement, :text)
```

Comment:
apologies to nitpick 😬 but could we remove the brackets from this to remain consistent with other migrations 🙏